_data/projects/histogramming/boost-histogram.yml

```diff
@@ -2,6 +2,6 @@ name: boost-histogram
 description: Python bindings for the C++14 Boost::Histogram library.
 url: https://github.com/scikit-hep/boost-histogram
 readme: https://github.com/scikit-hep/boost-histogram/blob/develop/README.md
-# docs: <url>
+docs: https://boost-histogram.readthedocs.io/en/latest/
```

Comment:
I forgot to ask: would it be possible to rather have the docs under scikit-hep.org/boost-histogram? We have that for root_numpy and I reckon it gives a nice sense of an ecosystem.